CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 ### capa explorer IDA Pro plugin
 
 ### Development
+- capa Explorer Web: improve navigation in capa Explorer Web
```

Comment:
let's add a new section `### capa Explorer Web` for these

Reply:
995d20ffb9199410c417bc4e72f5186e4f496609